src/logging.h

```diff
@@ -246,7 +239,7 @@ static inline bool LogAcceptCategory(BCLog::LogFlags category, BCLog::Level leve
 bool GetLogCategory(BCLog::LogFlags& flag, std::string_view str);
 
 template <typename... Args>
-inline void LogPrintFormatInternal(std::string_view logging_function, std::string_view source_file, const int source_line, const BCLog::LogFlags flag, const BCLog::Level level, util::ConstevalFormatString<sizeof...(Args)> fmt, const Args&... args)
+inline void LogPrintFormatInternal(std::string_view logging_function, std::string_view source_file, const int source_line, const BCLog::LogFlags flag, const BCLog::Level level, util::ConstevalFormatString<util::TrailingNewlineCheck, sizeof...(Args)> fmt, const Args&... args)
```

Comment:
nit: might be nice introducing a `LogFmt` alias here?

<details>
<summary>git diff on fa7190db83</summary>

```diff
diff --git a/src/logging.h b/src/logging.h
index e0bd79f793..ed224565e6 100644
--- a/src/logging.h
+++ b/src/logging.h
@@ -239,7 +239,10 @@ static inline bool LogAcceptCategory(BCLog::LogFlags category, BCLog::Level leve
 bool GetLogCategory(BCLog::LogFlags& flag, std::string_view str);
 
 template <typename... Args>
-inline void LogPrintFormatInternal(std::string_view logging_function, std::string_view source_file, const int source_line, const BCLog::LogFlags flag, const BCLog::Level level, util::ConstevalFormatString<util::TrailingNewlineCheck, sizeof...(Args)> fmt, const Args&... args)
+using LogFmt = util::ConstevalFormatString<util::TrailingNewlineCheck, sizeof...(Args)>;
+
+template <typename... Args>
+inline void LogPrintFormatInternal(std::string_view logging_function, std::string_view source_file, const int source_line, const BCLog::LogFlags flag, const BCLog::Level level, LogFmt<Args...> fmt, const Args&... args)
 {
     if (LogInstance().Enabled()) {
         std::string log_msg;
diff --git a/src/wallet/scriptpubkeyman.h b/src/wallet/scriptpubkeyman.h
index e038c4c77a..a16f5c99d6 100644
--- a/src/wallet/scriptpubkeyman.h
+++ b/src/wallet/scriptpubkeyman.h
@@ -254,7 +254,7 @@ public:
 
     /** Prepends the wallet name in logging output to ease debugging in multi-wallet use cases */
     template <typename... Params>
-    void WalletLogPrintf(util::ConstevalFormatString<util::TrailingNewlineCheck, sizeof...(Params)> wallet_fmt, const Params&... params) const
+    void WalletLogPrintf(LogFmt<Params...> wallet_fmt, const Params&... params) const
     {
         LogInfo("%s %s\n", m_storage.GetDisplayName(), util::RemoveSuffixView(tfm::format(wallet_fmt, params...), "\n"));
     };
diff --git a/src/wallet/wallet.h b/src/wallet/wallet.h
index f85bebe179..807c95ae1c 100644
--- a/src/wallet/wallet.h
+++ b/src/wallet/wallet.h
@@ -927,7 +927,7 @@ public:
 
     /** Prepends the wallet name in logging output to ease debugging in multi-wallet use cases */
     template <typename... Params>
-    void WalletLogPrintf(util::ConstevalFormatString<util::TrailingNewlineCheck, sizeof...(Params)> wallet_fmt, const Params&... params) const
+    void WalletLogPrintf(LogFmt<Params...> wallet_fmt, const Params&... params) const
     {
         LogInfo("%s %s\n", GetDisplayName(), util::RemoveSuffixView(tfm::format(wallet_fmt, params...), "\n"));
     };

```
</details>